__tests__/src/index.spec.ts

```diff
@@ -817,6 +852,29 @@ describe("flatpickr", () => {
       expect(fp.currentYear).toBe(2016);
     });
 
+    it("onKeyDown closes flatpickr on Enter when allowInput set to true", () => {
```

Comment:
This test is to ensure that this fix is covered:
https://github.com/flatpickr/flatpickr/pull/2419/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R1627